### ISSUE TITLE
Limit git history of opencv

### ISSUE THREAD

**Issue by y-a-v-a** [justadudewhohacks/npm-opencv-build#17]
Wouldn't it be an idea to limit the cloning of opencv by adding a limiting `--depth` parameter? See https://git-scm.com/docs/git-clone#git-clone---depthltdepthgt

**Comment by justadudewhohacks** [comment]
Good idea actually! That would probably allow the build to run faster.
